client/src/components/main/index.ts

```diff
@@ -0,0 +1 @@
+export * from './Main';
```

Comment:
Add new line at the end of the file (soon we'll have .editorconfig, which automatically does that).

Reply:
Done!